packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -552,6 +552,9 @@ export function CreateTaskForm({
                       id="priority"
                       label="Priority"
                       margin="normal"
+                      helperText={
+                        'Tasks that are not active with higher priority will be given precedence'
```

Comment:
Can we change the text to "A task with non-zero priority score will be given precedence"